pkg/model/local_target.go

```diff
@@ -15,6 +15,10 @@ type LocalTarget struct {
 	ignores   []Dockerignore
 
 	repos []LocalGitRepo
+
+	// Indicates that we should force this to run in parallel with other local
+	// resources, even if it's not safe to do so.
```

Comment:
```suggestion
	// resources (by default, this is presumed unsafe and is not allowed).
```

just to make this better communicate that it's not "we know it's not safe", but "we don't know it is safe"

Reply:
done